### PR TITLE
Enhance the README

### PR DESCRIPTION
**Pull request by satendrapalsingh**
Add standard README sections like Introduction, Installation, Usage, Contributing, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+
+These tutorials are designed to help you learn practical Chainblocks programming step-by-step. Each tutorial covers a few langauge features, blocks, and functions in the context of building a game from scratch.
+
+## Installation
+
+[Set up your environment](https://docs.fragcolor.xyz/contribute/code/getting-started/) and [build Chainblocks](https://docs.fragcolor.xyz/contribute/code/building-chainblocks/) before working through the tutorials given in this repository.
+
+## Usage
+
+Though there are no hard pre-requisites for these tutorials, if you're new to programming you should read the [Chainblocks primer](https://learn.fragcolor.xyz/how-to/chainblocks-primer/) first.
```
Comment (satendrapalsingh) [inline README.md:18]: Added TODO page for Chainblocks primer doc link with [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/c595e1fb6692c4d86b3e92096a39648a6307d516).

File: README.md [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+
+It's also recommended to go through the [Chainblocks architecture](https://docs.fragcolor.xyz/architecture-guides/chainblocks) guide to understand what's going on under the hood.
```
Comment (satendrapalsingh) [inline README.md:20]: Added TODO page for Chainblocks architecture guide link with [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/2db79b473ee45e88eeebd10a624e779c4616b6f1)

File: README.md [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
 
 * [Snake](./tutorials/snake/)
 
 *More to come. Stay tuned!*
 
+## Contributing
+
+Contributions in the form of new tutorials or fixes/enhancements to the existing ones are welcome! 
+
+Please refer to the [contribution guidelines](https://learn.fragcolor.xyz/how-to/contribute-tutorials/) for submitting your tutorials.
```
Comment (satendrapalsingh) [inline README.md:32]: Added TODO page for Contributing tutorials page link with [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/c595e1fb6692c4d86b3e92096a39648a6307d516).

File: README.md [hunk 4]
```diff
@@ -1,13 +1,36 @@
...
 
 * [Snake](./tutorials/snake/)
 
 *More to come. Stay tuned!*
 
+## Contributing
+
+Contributions in the form of new tutorials or fixes/enhancements to the existing ones are welcome! 
+
+Please refer to the [contribution guidelines](https://learn.fragcolor.xyz/how-to/contributing-tutorials/) for submitting your tutorials.
```
Comment (satendrapalsingh) [inline README.md:32]: Added TODO page for Contributing tutorials link with [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/c595e1fb6692c4d86b3e92096a39648a6307d516)

File: README.md [hunk 5]
```diff
@@ -1,13 +1,36 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
 
-## Tutorials
+This repository is a collection of example games and tutorials using the [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) programming language.
+
+Chainblocks is a lisp and hence is extremely powerful as well as fun to learn.
```
Comment (sinkingsugar) [inline README.md:8]: lisp inspired etc I'd say. it's not a lisp per se

File: README.md [hunk 6]
```diff
@@ -1,13 +1,36 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
```
Comment (Kryptos-FR) [inline README.md:4]: I would add a shield for the "learn" website that is built from this repository. Same as the one in the `chainblocks` repo.

```suggestion
[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
[![examples](https://img.shields.io/badge/learn-examples-blue)](https://learn.fragcolor.xyz/)
```

File: README.md (hunk 5 above)
Comment (satendrapalsingh) [reply README.md]: Fixed

File: README.md (hunk 6 above)
Comment (satendrapalsingh) [reply README.md:4]: Fixed

File: README.md [hunk 7]
```diff
@@ -1,13 +1,37 @@
...
+
+## Usage
+
+Though there are no hard pre-requisites for these tutorials, if you're new to programming you should read the [Chainblocks primer](https://learn.fragcolor.xyz/how-to/chainblocks-primer/) first.
+
+It's also recommended to go through the [Chainblocks architecture](https://docs.fragcolor.xyz/architecture-guides/chainblocks) guide to understand what's going on under the hood.
+
+### Tutorials
 
 * [Snake](./tutorials/snake/)
```
Comment (Kryptos-FR) [inline README.md:25]: I think we should link to the website page instead. here it opens the folder in GitHub but the markdown files are not meant to be read from there if someone wants to follow the tutorial.

```suggestion
* [Snake](https://learn.fragcolor.xyz/snake/)
```

Comment (satendrapalsingh) [reply README.md]: Ah! you're right - didn't notice that. Fixed now.

**Review by Kryptos-FR — APPROVED** [review]

**Pull request merged**